core/services/keystore/master.go

```diff
@@ -25,6 +25,8 @@ import (
 
 var ErrLocked = errors.New("Keystore is locked")
 
+type defaultEVMChainIDFunc = func() (defaultEVMChainID *big.Int, err error)
```

Comment:
Is this equals necessary? Is this technically declaring an alias to an anonymous type? (I think that's effectively equivalent anyways :shrug:)
```suggestion
type defaultEVMChainIDFunc func() (defaultEVMChainID *big.Int, err error)
```

Reply:
Did not know you could do that